doc/manual/packages.rst

```diff
@@ -574,7 +574,18 @@ going to have to do a *force push*:
 Creating a new Package
 ----------------------
 
-Guidelines for Naming a Package
+REQUIRE speaks for itself
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You should have a ``REQUIRE`` file in your package repo, with a bare minimum directive of what Julia version to include.
+As Julia moves to different versions, many packages include incorrect version requirements in their ``REQUIRE`` file. The most
+common mistake is to include pre-release versions of Julia after a release has been officially completed. If Julia has 
+officially released version ``0.3``, for example, then all the Julia-specific requirements in the ``REQUIRE`` file should not 
```

Comment:
Should we replace these with generic ``0.x`` and ``0.y`` so we don't have to keep updating them?